docs/guide/seo.rst

```diff
@@ -0,0 +1,57 @@
+================================
+Search Engine Optimization (SEO)
+================================
+
+What is search engine optimization?
+-----------------------------------
+
+Search engines drive a significant portion of site traffic for docs that publish as public-facing websites.
+Search engine optimization (SEO) describes the means and methods used to have your web pages display for a set of desired search terms.
+
+The work of SEO demands familiarity with the ever-changing rules of the algorithms search engines use to select and arrange web sites to return for any particular query.
+These algorithms change frequently, so the requirements for achieving the best optimization also change frequently.
+
+Fortunately, good tech writing already does most of the standard practices required for excellent SEO.
+
+- Write good headings
+- Make your docs skimmable by breaking content into sections under appropriate headings
+- Focus on good information architecture that provides quality navigation and site hierarchy
+- Include page metadata
```

Comment:
where? and can we provide examples (like last updated date?)

Reply:
How about
```
- Include page metadata, such as last modified date, in the `<head>` section of your web page (how to do this varies by your publication toolchain)
```

Reply:
let me be more constructive:

```suggestion
- Make sure your webpages include metadata, such as that described in the `The Ultimate Guide to SEO Meta Tags <https://moz.com/blog/the-ultimate-guide-to-seo-meta-tags>`__ by Moz.
```

Reply:
Jinx comments! I think your suggestion is excellent!

Reply:
Gonna drop that Moz link into the blog list, though!